guide.md

```diff
@@ -39,32 +39,30 @@ Illustration sources
 
-This guide was built on that very idea. Like a travel guide, it provides an introduction to a “foreign place”, shows you the highlights and tells you some of the best stories. It provides facts and maps to figure out how to get around, ideas for tours and things to do. Whether you end up taking the official museum tour, clubbing all night or go on shopping sprees is still your choice. Note that these are guides, not mandates. Just like in a foreign city, you can choose to go off the path, and forge a new direction. Others may just follow in the future.
+We designed this guide to help anyone navigate those complexities—regardless of where they live or work. We can’t imagine every possible use case, but we can help you build for them.
 
 ## What's in it?
 
-With Bitcoin being so new and different, a big focus of the guide lies on providing overviews, explanations and best practices for the most common questions and problems. These are the main chapters:
+The Bitcoin Design Guide is divided into the following sections:
 
-- [Getting started]({{ '/guide/getting-started/introduction/' | relative_url }}) is for top-level overviews, whether it's bitcoin technology, visual language, software, or design collaboration in an open-source world
-- [Onboarding]({{ '/guide/onboarding/introduction/' | relative_url }})  goes in-depth on the delicate process of introducing new users to bitcoin and how to use specific products and features
-- [Private key management]({{ '/guide/private-key-management/introduction/' | relative_url }}) provides extensive details and guidance on the many techniques for storing private keys
-- [Payments]({{ '/guide/payments/' | relative_url }}) deals with the complexities of sending and receiving bitcoin across various use cases
-- [Foundations]({{ '/guide/foundations/' | relative_url }}) highlights principles, best practices and other useful information for designing Bitcoin products
-- [Case studies]({{ '/guide/case-studies/' | relative_url }}) explores approaches to diverse uses cases with wireframes, designs and prototypes.
-- [Glossary]({{ '/guide/glossary/' | relative_url }}) explains common Bitcoin terminology, from Address to ZPUB.
+- [Getting started]({{ '/guide/getting-started/introduction/' | relative_url }}) provides top-level overviews of common visual language, software, and best practices for bitcoin design collaboration
+- [Onboarding]({{ '/guide/onboarding/introduction/' | relative_url }}) introduces new users to must know bitcoin terms, products, and features
+- [Private key management]({{ '/guide/private-key-management/introduction/' | relative_url }}) details the many techniques that can be used to store private keys
+- [Payments]({{ '/guide/payments/' | relative_url }}) covers the complexities encountered when sending and receiving bitcoin
+- [Foundations]({{ '/guide/foundations/' | relative_url }}) highlights principles and other fundamentals designing Bitcoin products
```

Comment:
```suggestion
- [Foundations]({{ '/guide/foundations/' | relative_url }}) highlights principles and other fundamentals to keep in mind when designing Bitcoin products
```
Broken sentence.

Reply:
Can you elaborate?

Reply:
The original text is: `Foundations highlights principles and other fundamentals designing Bitcoin products.`

Reads like the principles and fundamentals are entities that design Bitcoin products.

Reply:
Or with less words 

```suggestion
- [Foundations]({{ '/guide/foundations/' | relative_url }}) highlights principles and other fundamentals of designing Bitcoin products
```